instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/ClientAddressAndPortExtractor.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.instrumenter.http;
+
+import static io.opentelemetry.instrumentation.api.instrumenter.http.ForwardedHeaderParser.extractClientIpFromForwardedForHeader;
+import static io.opentelemetry.instrumentation.api.instrumenter.http.ForwardedHeaderParser.extractClientIpFromForwardedHeader;
+
+import io.opentelemetry.instrumentation.api.instrumenter.network.internal.FallbackAddressPortExtractor;
+
+final class ClientAddressAndPortExtractor<REQUEST>
```

Comment:
New class missing unit test.

Reply:
TBD in https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/9009